src/components/TradeWidget/TokenRow.tsx

```diff
@@ -198,15 +217,18 @@ const TokenRow: React.FC<Props> = ({
             pattern: { value: validInputPattern, message: 'Invalid amount' },
             validate: { positive: validatePositive },
           })}
-          onKeyPress={preventInvalidChars}
+          onKeyPress={onKeyPress}
           onChange={enforcePrecision}
           onBlur={removeExcessZeros}
+          tabIndex={tabIndex + 2}
         />
         {errorOrWarning}
         <WalletDetail>
           <div>
             <strong>
-              <LinkWithPastLocation to="/deposit">Exchange wallet:</LinkWithPastLocation>
+              <LinkWithPastLocation to="/deposit" tabIndex={-1}>
```

Comment:
Why `tabIndex={-1}` instead of `0`?

Reply:
Because I wanted to not allow this to be tab-indexable. Should it be?

Reply:
Then it's all good.